backend/python/app/services/implementations/story_service.py

```diff
@@ -834,6 +835,28 @@ def remove_user_from_story_translation(self, story_translation_id, user_id):
             self.logger.error(error)
             raise error
 
+    def get_story_translation_statistics(self):
```

Comment:
make sure you dont count tests! ie, add `.filter(StoryTranslation.is_test == False)`